### ISSUE TITLE
[Todo] Can add container utilities

### ISSUE THREAD

**Issue by zaydek** [zaydek/duomo#27]
These should be simple enough to implement: https://tailwindcss.com/docs/container#app.

**Comment by zaydek** [comment]
We already have container classes implicitly via `max-w`.

**Comment by zaydek** [comment]
```
.container {
  width: 100%;
}

@media (min-width: 640px) {
  .container {
    max-width: 640px;
  }
}

@media (min-width: 768px) {
  .container {
    max-width: 768px;
  }
}

@media (min-width: 1024px) {
  .container {
    max-width: 1024px;
  }
}

@media (min-width: 1280px) {
  .container {
    max-width: 1280px;
  }
}

@media (min-width: 1536px) {
  .container {
    max-width: 1536px;
  }
}
```

**Comment by zaydek** [comment]
Implemented for 0.5.1 or 0.6.0.
